searchcore/src/tests/proton/documentmetastore/documentmetastore_test.cpp

```diff
@@ -1044,6 +1044,9 @@ struct SplitAndJoinEmptyFixture {
     void assertNotifyCreateBuckets(std::vector<document::BucketId> expBuckets) {
         EXPECT_EQ(expBuckets, _bucketCreateListener._buckets);
     }
+    void assertBucketDB() {
```

Comment:
It's not immediately obvious what this asserts, consider renaming to `assertBucketDBIsConsistent` or something down that line

Reply:
Fixed